src/main/java/se/sundsvall/notes/api/model/Event.java

```diff
@@ -0,0 +1,112 @@
+package se.sundsvall.notes.api.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import java.util.Objects;
+
+import static io.swagger.v3.oas.annotations.media.Schema.RequiredMode.REQUIRED;
+
+public class Event {
+
+	@Schema(description = "Operation in event", example = "replace", requiredMode = REQUIRED)
```

Comment:
Should we have requiredMode here? Isn't this atrribuute only applicable for write-operations?

(FYI: for delete-operations the, value will be omitted, by default)